packages/lit-dev-content/site/tutorials/content/svg-templates/05.md

```diff
@@ -0,0 +1,137 @@
+This section of the tutorial covers the basics of using CSS with SVG 
+including how to:
+
+- Style SVG with CSS
+- Use CSS Custom Properties to theme SVG
+
+### Learn
+
+CSS can apply presentation attributes to SVG elements similar to how 
+CSS applies attributes to HTML.
+
+However, only [SVG presentation attributes](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/Presentation)
+can be applied to SVG through CSS.
+
+In the following example, all `<rect>` elements will be as wide as
+their parent element while elements using the class `background` will
+have a black fill.
+
+
+```ts
+const helloSvgCss = css`
+	rect {
+		width: 100%;
+	}
+
+	.background {
+		fill: #000;
+	}
+`;
+```
+
+CSS classes can be applied to SVG Elements like in the example below.
+
+```ts
+const helloCssClasses = html`
+	<rect class="background"></rect>
+`;
+```
+
+SVG can also be styled with CSS Custom Properties. This enables artists
+and designers to theme an SVG document with the same styles as its
+HTML counterpart.
+
+```ts
+const helloCssCustomProperties = css`
+	.background {
+		fill: var(--background-color, #ff8800);
+	}
+`;
+```
+
+### Apply
+
+Add any presentation attributes to a CSS template like the example
+below.
+
+In the example below, properties for `svg`, `text`, and `rect` elements
+have been reimplemented as CSS. What's left are elements with `id`
+selectors and transformations.
```

Comment:
This doesn't suggest what to do with the snippet. Tell the user to put it in their code but it shouldn't change anything until they `static styles = [...]`

Reply:
sounds good! done :)